src/jira/client/jira-client.ts

```diff
@@ -237,17 +243,12 @@ export const getJiraClient = async (
 							truncatedPRs: getTruncatedIssuekeys(data.pullRequests)
 						}, issueKeyLimitWarning);
 						truncateIssueKeys(data);
-						const subscription = await Subscription.getSingleInstallation(
-							jiraHost,
-							gitHubInstallationId
-						);
-						await subscription?.update({ syncWarning: issueKeyLimitWarning });
 					}
 
 					return batchedBulkUpdate(
 						data,
 						instance,
-						gitHubInstallationId,
+						subscription?.gitHubInstallationId || gitHubInstallationId,
```

Comment:
Can you please explain why `subscription?.gitHubInstallationId || gitHubInstallationId,`?

Reply:
`getSingleInstallation` either returns an entry or `null`. In the case where a subscription is found, we want to set the installationId in the payload to the subscription's gitHubInstallationId. However, because it possibly returns `null`, if we don't have a default (`|| gitHubInstallationId`) TS complains:

![Screen Shot 2022-08-12 at 3 37 59 pm](https://user-images.githubusercontent.com/37155488/184291485-03a0ba8f-09dd-4b55-b63d-1c9c163fd6c5.png)

Reply:
`In the case where a subscription is found, we want to set the installationId in the payload to the subscription's gitHubInstallationId` <-- which will be the exact value of `gitHubInstallationId`? So the origin code 
```
return batchedBulkUpdate(
						data,
						instance,
						gitHubInstallationId,
```
will still work?